README.md

```diff
@@ -1,251 +1,395 @@
+NSDictionary *keyAndValue   = @{ key : value };
 
-```objc
-NSDictionary *contactDictionary = @{ @"Marc Bevilaqua": @2125551212, 
-									    @"Jim Sherman": @8581119999,
-									 @"Lena Ricciardi": @9736661111 };
+NSDictionary *keysAndValues = @{ key1 : value1, 
+                                 key2 : value2 };
 ```
 
-Now let's attempt a more advanced `NSDictionary` using the same example. In a real contact list, each of our contacts would have a phone number, an email address, and a favorite color. In this case, each contact would have its own `NSDictionary` of attributes!
-
-For example:
+The dictionary literal employs the curly braces (`{``}`) to wrap its list of key-value pairs which are separated from each other in the list using `,`s ("commas"); the keys and values themselves are separated from each other with a `:` ("colon") with the key on the left and the value on the right.
 
-```objc
-NSDictionary *marcBevilaquaDictionary = { @"Phone Number": @2125551212, 
-									     		 @"Email": @"me@marcbevilaqua.com",
-											     @"Color": @"Orange" };
+**Top-tip:** *While there are some limits on which objects can be keys, it's most common to use strings. Values can be of any object type, even other collections.* 
 
-NSDictionary *jimShermanDictionary = { @"Phone Number": @8581119999, 
-											  @"Email": @"me@jimsherman.com",
-											  @"Color":@"Blue" };
+Translating our `jenny` array into a dictionary with this syntax would look like this:
 
-NSDictionary *lenaRicciardiDictionary = { @"Phone Number": @9736661111, 
-												 @"Email": @"me@lenaricciardi.com",
-												 @"Color": @"Seafoam" };
+```objc
+//                            KEYS               VALUES
+NSDictionary *jenny = @{ @"first_name"       : @"Jenny",
+                         @"relationship"     : @"Friend",
+                         @"phone_number"     : @"(555) 867-5309",
+                         @"email_address"    : @"jenny@email.com",
+                         @"physical_address" : @"123 Street Name",
+                         @"city_state"       : @"Anywhere, USA",
+                         @"zip_code"         : @"00409"        };
+
+NSLog(@"%@", jenny);
+```
+This might print:
 
-NSDictionary *contactDictionary = @{ "Marc Bevilaqua": marcBevilaquaDictionary, 
-									   @"Jim Sherman": jimShermanDictionary, 
-								    @"Lena Ricciardi": lenaRicciardiDictionary };
 ```
+{
+    "city_state" = "Anywhere, USA";
+    "email_address" = "jenny@email.com";
+    "first_name" = Jenny;
+    "phone_number" = "(555) 867-5309";
+    "physical_address" = "123 Street Name";
+    relationship = Friend;
+    "zip_code" = 00409;
+}
+```
+Did you notice how the order changed when it printed? That's because Apple does ***not*** guarantee that a dictionary will maintain the order of its components (you should always expect that it won't). What it *does* guarantee, however, is the key-value binding. Because of this, dictionaries do not respond to indexing.
```

Comment:
also following the dictionary metaphor. When the dictionary adds a word...they never guarantee the fact that the 234th word is apple. But they do guarantee that to get the definition of apple, you look up apple